deploy/stacks/frontend_cognito_config.py

```diff
@@ -105,6 +105,7 @@ def __init__(
                 'envname': envname,
                 'deployment_region': backend_region,
                 'custom_domain': str(bool(custom_domain)),
+                'redeploy': 'true',
```

Comment:
Unfortunately this will only work the first time you deploy it and then it will stop finding diffs and won't redeploy.
The env variables solutions is to pass something random to an env variable, i.e timestamp or uuid but this will always make CDK return a diff which isn't healthy.